Abstraction/Program.cs

```diff
@@ -1,31 +1,22 @@
-﻿namespace Vsite.Oom.ObjectModel
+﻿try
 {
-    internal class Program
```

Comment:
Izbrisali ste zaglavlje koda (tj. ovaj file Vam mora početi s):
```
namespace Vsite.Oom.ObjectModel
{
    internal class Program
    {
        static void Main(string[] args)
        {
```
Na kraju trebate onda dodati  3 vitičaste zagrade:
```
        }
    }
}
```
Kada to popravite stisnite Ctrl+K, Ctrl+D (ili iz menia Edit - Advanced izaberete "Format Document") da se kod preformatira i bude pregledniji.